NorthstarLauncher/main.cpp

```diff
@@ -147,7 +187,7 @@ void EnsureOriginStarted()
 		return;
 	}
 
-	printf("[*] Starting Origin...\n");
```

Comment:
Any particular reasons we're switching all the prints to `printf` in this file? ^^

Reply:
Nevermind, got it wrong, we switched **from** `printf`s to C++ style prints.

Reply:
reason is, "cause we're in C++, not C"